src/ui/Facet/ValueElementRenderer.ts

```diff
@@ -58,15 +58,17 @@ export class ValueElementRenderer {
   }
 
   protected buildExcludeIcon(): HTMLElement {
+    const isExcluded = this.facetValue.excluded;
     const excludeIcon = $$('div', {
-      title: l('ExcludeValueWithResultCount', this.caption, l('ResultCount', this.count, parseInt(this.count, 10))),
+      ariaLabel: l('ExcludeValueWithResultCount', this.caption, l('ResultCount', this.count, parseInt(this.count, 10))),
```

Comment:
I'm still uncertain whether we should keep the `title`. Titles, as opposed to labels, show a tooltip on hover and *could* be helpful.

However, there aren't many places in JSUI where a `title` is prioritized. Additionally, a title would be likely be expected to change by most users since the exclude button doesn't *look* like a checkbox, but would likely be expected not to change by users who use accessibility tools since the exclude button would be represented as a checkbox in the accessibility tree.

An alternative could be to have both a `title` and `aria-label`, but [screen readers behave very differently](https://www.deque.com/blog/text-links-practices-screen-readers/#post-13282:~:text=The%20Comparison) when that's the case.

Reply:
Title helps the average user understand what the button is doing. The way the exclude button is displayed, it can be a bit confusing at first.

Reply:
Good point. Recently, one of our clients misunderstood the purpose of the exclude button and mistakenly reported an issue on it before finding out and correcting themselves.

I'll use the `title` instead of `aria-label`.